test.md

```diff
@@ -0,0 +1 @@
+"This file should be ignored by git"
```

Comment:
If you want a file to be ignored by git remember to add it to [your `.gitignore`](https://github.com/Vianney992/mobile-version/blob/1e68df02458a42658953f571a49979f2025696b8/.gitignore) file :wink: